### ISSUE TITLE
userpools and focuswallet

### ISSUE THREAD

**Issue by whitindllc** [RainbowMiner/RainbowMiner#2691]
i have 4 or 5 user pools setup and when i try to use the focuswallet to mine to them none of them work....

**Comment by RainbowMiner** [comment]
This sounds like a bug. I will have a look.

**Comment by whitindllc** [comment]
i should preface that focuswallet works fine with predefined pools.... once you have built in

you helped me build 1 awhile ago for anon and that one worked... i just didnt use it in a long time and then when i started adding new ones none of them worked... including the first one...

**Comment by whitindllc** [comment]
any luck replicating this or is this just a me issue?

**Comment by jeromemorignot** [comment]
> any luck replicating this or is this just a me issue?

I found a solution to this. I had the exact same issue. You need to edit pools.config.txt and add the coin to the list of coins under the pool entry in this file.

**Comment by whitindllc** [comment]
Ill play with this tonight.... 

when you put coinsymbol does it just automatically mine this and only this... or do you need to go to the focuswallet and add it

wonder what enableautocoin does

**Comment by jeromemorignot** [comment]
> Ill play with this tonight....
> 
> when you put coinsymbol does it just automatically mine this and only this... or do you need to go to the focuswallet and add it
> 
> wonder what enableautocoin does

For userpools you need one entry per coin/stratum so I would say - it will mine only this. I havent tried it though.
enableautocoin is supposed to add the coins you have setup  a wallet for in RB automatically in the pool. It may redundant with adding it at the end of the file...I wanted to make sure MEWC would be mined :P

**Comment by RoyCRoderickIII** [comment]
Agreed the few userpools i have added have been for a single coin, and it mines them just fine.

**Comment by whitindllc** [comment]
that worked.... i just found the entry in the pools and noticed that the coin field had no address... so i gave it an address and focuswallet to it and it worked fine.... interesting
